addon/globalPlugins/nao.py

```diff
@@ -32,13 +34,25 @@
 pdfToPngToolPath = "\""+os.path.join (addonPath, "tools", "pdftopng.exe")+"\""
 pdfToImagePath = "" + os.path.join (addonPath, "images") + ""
 pdfToImageFileNamePath = pdfToImagePath + "\\img"
+ADDON_SUMMARY = addonHandler.getCodeAddon().manifest["summary"]
 
 class GlobalPlugin(globalPluginHandler.GlobalPlugin):
+	try:
+		scriptCategory = unicode(ADDON_SUMMARY)
+	except NameError:
+		scriptCategory = str(ADDON_SUMMARY)
+
 	def __init__(self, *args, **kwargs):
 		super().__init__(*args, **kwargs)
 		self.recogUiEnhance = recogUiEnhance.RecogUiEnhance()
 		self.beeper = beepThread.BeepThread()
 
+
+	@script(
+		# Translators: Message presented in input help mode.
+		description=_("NAO: take a full screen shot and recognize it."),
```

Comment:
I think there is no need to add "NAO" here since it's already the name of the category in the input gesture dialog.
```suggestion
		description=_("Take a full screen shot and recognize it."),
```